cluster_config/steps.yaml.tpl

```diff
@@ -15,6 +15,9 @@ BootstrapActions:
 - Name: "metrics-setup"
   ScriptBootstrapAction:
     Path: "s3://${s3_config_bucket}/component/analytical-dataset-generation/metrics-setup.sh"
+- Name: "download-pdm-sql"
```

Comment:
it's PDM sql right? Should it be mongo latest? Just causes confusion as we already have PDM

Reply:
corrected the name